### DISCUSSION TITLE
Zapytanie

### DISCUSSION THREAD

**Discussion by Belfer79** [arkypita/LaserGRBL#1125] [Q&A]
Dzieki waszym instrukcjam udało mi sie skonfigurowac laser grbl ma jednak lekki problem po wypaleniu obrazka jego boczne krawedzie są jakby bardziej przypalone Ttak jakby laser przy zmianie kierunku  bardziej przypalal jak to moge zniwelowac

**Comment by StuartB4** [comment]
Czy używasz M4 w ustawieniach obrazu docelowego. Jeśli nie, zmień go na M4 za pomocą menu rozwijanego.

![Untitle999d](https://user-images.githubusercontent.com/16980026/93602385-73c8e380-f9ba-11ea-8397-6a0ca808c776.jpg)

**Comment by Belfer79** [comment]
Dziekuje pomogło

**Comment by StuartB4** [comment]
Przepraszam, nie rozumiem, co masz na myśli przez środkową linię?
